netfoundry.platform/galaxy.yml

```diff
@@ -8,7 +8,7 @@ namespace: netfoundry
 name: platform
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.19.0
+version: 1.19.1
```

Comment:
Bump the major version to indicate to the user that there may be a compatibility issue.